modules/utxo-lib/package.json

```diff
@@ -49,14 +49,15 @@
     "@bitgo/blake2b": "^3.0.3",
     "@noble/secp256k1": "1.6.3",
     "bip32": "^3.0.1",
+    "bip174": "npm:@bitgo/bip174@3.0.0",
     "bitcoin-ops": "^1.3.0",
-    "bitcoinjs-lib": "npm:@bitgo/bitcoinjs-lib@7.0.0-rc.0",
+    "bitcoinjs-lib": "git+https://github.com/BitGo/bitcoinjs-lib#3876701401d62a332c4901aa2a6f6b71c3cd799d",
     "bn.js": "^5.2.1",
     "bs58check": "^2.1.2",
     "cashaddress": "^1.1.0",
     "create-hash": "^1.2.0",
     "create-hmac": "^1.1.7",
-    "ecpair": "^2.0.1",
+    "ecpair": "git+https://github.com/brandonblack/ecpair#95667235852b2c28565012c292d6c9cb3b2eee38",
```

Comment:
Is there a todo to revert when this PR is merged and published? https://github.com/bitcoinjs/ecpair/pull/11

Reply:
Yep, we're discussing the options there, as the maintainer has been idle for some months. May end up having to fork off a BitGo fork.